custom_components/tuya_local/devices/caldo_up_t_wifi.yaml

```diff
@@ -4,20 +4,18 @@ products:
     name: Olimpia Splendid Caldo Up T
 primary_entity:
   entity: climate
-  translation_only_key: heater
   dps:
     - id: 1
       type: boolean
       name: hvac_mode
       mapping:
         - dps_val: true
-          value: heat
+          value: "heat_cool"
```

Comment:
This is a heater, there is no reason to change this from heat to heat_cool, as it is not capable of automatically switching between heating and cooling.